containers/dashboardEdit/EditDashboardName/EditDashboardName.module.scss

```
@@ -0,0 +1,53 @@
+@use '@/styles/index.scss' as *;
+
+.container {
+  @include Flex(column, null, null);
+  @include ResponsiveWidth(620px, 544px, 284px);
+  @include ResponsivePadding(32px 28px, 32px 28px, 20px 16px);
+  @include border-radius(16px);
+  .section-header-title {
+    @include FontStyle('2xl', 'bold', '2xl', 'xl');
```

질문이 있는데 이렇게 2xl를 스트링으로 넣으면 적용이 안되지않나요?! 
전 저렇게 넣으면 2xlpx로 적용되더라구요 그래서 한 번 확인해주시면 좋을거 같습니다!! 
안되시면 싱글쿼터 빼주시면 될거같고 되면 그냥 진행해주시면 될거같습니다!!
```suggestion
@include FontStyle(2xl, bold, 2xl, xl);
```